### PR TITLE
feat(export): Export Copyrights

### LINKED ISSUE #1539: Include OJO results in "License List" tab of fossology
#### Description

OJO findings are missing in "License List".

Thanks to @maxhbr for pointing this out.

### LINKED ISSUE #1687: Export list of files with copyright statement but without license statement.
### Description

It will be very helpful if fossology tool provides option to export without license files path 

#### Screenshots

IMG_20200417_122807

### PR DESCRIPTION
**Pull request by GMishx**
## Description

Added new functionality for exporting copyrights to License List. Thus renamed the page to more general Export List.

The page can now export License List (as before) and Copyright List as well. The Copyright List has a filter to export all copyrights or to export copyrights from files which do not have license.

Files without license here refers to:
1. File had no license finding by agents and no license was added by users.
2. File had license findings by agents but were either removed or file was marked as irrelevant.

### Changes

1. Rename page License List to Export List and file `src/www/ui/ui-license-list.php` to `src/www/ui/ui-export-list.php`.
1. Rename template `src/www/ui/template/ui-license-list-form.html.twig` to `src/www/ui/template/ui-export-list.html.twig`.
1. Add copyright exports to `UIExportList`.
1. Fix the links in UI.

## How to test

1. Upload a package without running any agents and check if "Export List" can handle it.
1. Run only copyright agent on the package and do some manual clearing.
    1. All the copyrights should be exported for both options.
1. Run some scanners and do some clearing.
    1. The copyright list should reflect the changes when exporting files without licenses.

## Todo
Need new issues for following:
- [x] Add copyright export to `fo_nomos_license_list`. (#1715)
- [x] Add copyright export to REST API. (#1716)

Closes #1687
Closes #1539

### CODE REVIEW

File: src/www/ui/template/ui-export-list.html.twig [hunk 1]
```diff
@@ -0,0 +1,160 @@
+{# Copyright 2016-2017 Siemens AG
```
Comment (shaheemazmalmmd) [inline src/www/ui/template/ui-export-list.html.twig:1]: Please change copyright years

Comment (GMishx) [reply src/www/ui/template/ui-export-list.html.twig]: Updated, please check.

**Comment by shaheemazmalmmd** [comment]
Duplicate copyrights are reported in CSV file. UI looks fine please see the screen shot.

screenShot17

**Comment by GMishx** [comment]
I have created another issue for the duplicate copyrights as it is coming from the library itself: https://github.com/fossology/fossology/issues/1742

**Comment by GMishx** [comment]
Hello @shaheemazmalmmd, sorry, the SPDX agent made me believe that the library has issues. I reffered to other reports and figured how to fix it.

So, I've pushed the fix, please check now.

**Comment by DineshDevarajan** [comment]
@GMishx is Export Copyrights available as a part of REST API ?

**Pull request merged**